src/h5web/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -42,9 +44,24 @@ function Breadcrumbs(props: Props) {
           />
         );
       })}
-      <span className={styles.crumb} data-current>
-        {crumbs[crumbs.length - 1]}
-      </span>
+      <button
+        className={styles.crumbButton}
+        type="button"
+        title="Copy path to clipboard"
+        onClick={() => {
+          navigator.clipboard.writeText(path);
+          setPathCopied(true);
+        }}
+        onMouseLeave={() => setPathCopied(false)}
+        data-current
+      >
+        <span className={styles.crumb}>{crumbs[crumbs.length - 1]}</span>
+        {isPathCopied ? (
+          <FiCheck className={styles.copyIcon} />
+        ) : (
+          <FiClipboard className={styles.copyIcon} />
+        )}
```

Comment:
You can store the icon in a constant like `const CopyIcon = isPathCopied ? FiCheck : FiClipoard` and then use it here `<CopyIcon className... />`